src/navigation/navigationstate.cpp

```diff
@@ -89,11 +89,39 @@ NavigationState::NavigationState(const ghoul::Dictionary& dictionary) {
     referenceFrame = p.referenceFrame.value_or(anchor);
     aim = p.aim.value_or(aim);
 
-    if (p.up.has_value()) {
-        up = *p.up;
+    up = p.up;
+    yaw = p.yaw.value_or(yaw);
+    pitch = p.pitch.value_or(pitch);
+}
+
+NavigationState::NavigationState(const nlohmann::json& json) {
+    position.x = json["position"]["x"].get<double>();
+    position.y = json["position"]["y"].get<double>();
+    position.z = json["position"]["z"].get<double>();
+
+    anchor = json["anchor"];
+
+    if (auto it = json.find("referenceframe");  it != json.end()) {
+        referenceFrame = it->get<std::string>();
```

Comment:
This should be set to the anchor node if the reference frame was not specified

Reply:
Currently, loading a navigation stat ethat looks like this leads to an error:
```
{
  "anchor": "Earth",
  "position": {
    "x": 13514805.101321544,
    "y": 5454659.039307959,
    "z": -18279333.341204002
  },
  "up": {
    "x": 0.8035739201794957,
    "y": 0.0035944893626913277,
    "z": 0.59519411493527
  }
}
```

![image](https://github.com/OpenSpace/OpenSpace/assets/8808894/d1c26b97-488f-4854-939a-c2581e7c3f01)